routes/DeleteProductByID.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by Kun on 7/5/2015.
+ */
+
+
+var express = require("express");
+var router = express.Router();
+var Q = require('q');
+
+
+router.route("/:productID").get(function(req,res){
+    Q.longStackSupport = true;
+    var db = require("../imp_services/impdb.js").connect();
+    Q.fcall(db.beginTransaction())
+        .then(db.query("USE " + db.databaseName))
+        .then(db.query("CALL DeleteProductByID(" + req.params.productID+");"))
+        .then(function(rows){
+            console.log("Success");
```

Comment:
can we output something to the console that is more meaningful than "success"? Maybe something like "Deleted product <productID> successfully"?